opencensus/exporters/stats/prometheus/README.md

```diff
@@ -0,0 +1,56 @@
+# OpenCensus Prometheus Stats Exporter
+
+The Opencensus Prometheus Stats Exporter is a stats exporter that exposes stats
+to the
+[Prometheus cpp client library](https://github.com/jupp0r/prometheus-cpp).
+
+## Quickstart
+
+### Prerequisites
+
+The Prometheus exporter exposes stats on an HTTP endpoint that is scraped by
+a Prometheus server. To install and run the Prometheus server, follow
+the instructions [here](https://prometheus.io/docs/introduction/first_steps/).
+
+### Register the exporter
+
+#### Using the Prometheus client library's exposer
+
+Register the Exposer on a selected port, and then register the Opencensus
```

Comment:
Register when? Before / after configuring views? Maybe hint to do it "in main" or "before starting the RPC server" or whatever the binary is doing?